04_numeronym/patrickmarabeas/main_test.go

```diff
@@ -0,0 +1,80 @@
+package main
+
+import (
+	"bytes"
+	"fmt"
+	"testing"
+)
+
+func TestMainOutput(t *testing.T) {
+	var buf bytes.Buffer
+	out = &buf
+
+	main()
+
+	expected := fmt.Sprintf("%s\n", "[a11y K8s abc]")
+	got := buf.String()
+
+	t.Run("Main function", func(t *testing.T) {
+		if expected != got {
```

Comment:
Consider using testify package  to validate test cases `"github.com/stretchr/testify/assert"`. Its stressed by the team. But, will leave this upto you.

Reply:
I'll look into it for future labs, thanks.